src/ConstructiveSolidGeometry/VolumePrimitives/Cone.jl

```diff
@@ -83,22 +83,31 @@ struct Cone{T,CO,TR,TP<:Union{Nothing,T}} <: AbstractVolumePrimitive{T, CO}
 end
 
 #Type conversion happens here
+function Cone{T,CO}(r, φ::Tuple{T,T}, hZ, origin, rotation) where {T,CO}
+    r = _csg_convert_args(T, r)
+    φ = _csg_convert_args(T, φ)
```

Comment:
If you force `φ` to be `Tuple{T,T}`, I don't see why you would need to `_csg_convert_args` it afterwards..